README.md

```diff
@@ -50,6 +50,17 @@ points to the location of the Android SDK:
 
 1. A Mac with MacOS X 10.13.2 or superior
 2. Xcode 9.2 or superior, available from the Mac App Store
+3. The `usbmuxd` library, a cross-platform software library that talks the protocols to support iOS devices, is required. Install it with the use of [Homebrew](https://brew.sh):
```

Comment:
... is required in case you want to install your app on iOS Devices (it is not required if you want to run in the Simulator)